luskan/src/main/resources/PluginFactory.properties

```diff
@@ -0,0 +1,26 @@
+
+## commented out values are the defaults, shown as examples
+## to customise behaviour, subclass the specified class and 
+## change the configuration here
+
+# configure supported values of LANG parameter and the TapQuery implementation
+ca.nrc.cadc.tap.TapQuery.langValues = ADQL ADQL-2.0
+ADQL =     org.opencadc.luskan.impl.AdqlQueryImpl
+ADQL-2.0 = org.opencadc.luskan.impl.AdqlQueryImpl
+
+# TBD - ca.nrc.cadc.tap.MaxRecValidator
```

Comment:
If you are going to keep commented out settings, the value should be the default, not a classname from someplace else that is always going to be wrong. For example, the TableWriter one below is the default value so removing the comment char doesn't actually change anything; that's a good way to do it.

Reply:
I don't know all the defaults so I left the hooks for the plugings